<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>611</width>
     <height>651</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="position_x_text">
        <property name="text">
         <string>Position X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="position_x_line"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="position_z_text">
        <property name="text">
         <string>Position Z</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="position_z_line"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="position_y_text">
        <property name="text">
         <string>Position Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="position_y_line"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="update_pivot_position_button">
        <property name="text">
         <string>Update Pivot Position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="set_pivot_point_button">
        <property name="text">
         <string>Set Pivot Point</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rest_pivot_point_button">
        <property name="text">
         <string>Rest Pivot Point</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
